arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -663,6 +662,9 @@
             "schema": {
               "$ref": "#/definitions/RedisPatchSchedule"
             }
+          },
+          "404": {
+            "description": "Response of get patch schedules if it does not exist."
```

Comment:
Be aware that this prevents exceptions from being thrown on 404 (instead, `null` will be returned). Is this the intended behavior?